core/controllers/payload_validator.py

```diff
@@ -81,9 +81,16 @@ def validate_arguments_against_schema(
     errors = []
     # Dictionary to hold normalized values of arguments after validation.
     normalized_values = {}
-
     for arg_key, arg_schema in handler_args_schemas.items():
-
+        if (
+                arg_key in handler_args and
```

Comment:
Hi @sagangwee, just to check -- I thought for request params the idea was also not to include this key at all in the request? I.e. in general if the key had a value of None then we shouldn't even pass that key-value pair?

Note that Vojta suggested this change in addition to the above, I believe (from reading his comment).

Reply:
Ah, gotcha. I thought Vojta was referring to the PR change. Done.

Reply:
But I still see keys with a value of None in the request... (e.g. in the test files above).

I have a suggestion though. Instead of doing this type conversion here, let's drop it and instead prevent None/null values from being sent in the first place. To do so:

(a) modify RequestInterceptor and $httpInterceptor in the frontend to throw an error if a null value is sent as part of the param dict
(b) modify this code to throw an error if a null value is sent as part of a get/delete request

Would that be better? That way we avoid the wrong things being sent in the first place, rather than doing a potentially ambiguous type conversion. /cc @vojtechjelinek @Nik-09

Reply:
Yeah, I agree it would be better to handle this in the frontend.

Reply:
@seanlip Ah, I see. Updated.